tests/L0/run_transformer/test_pipeline_parallel_fwd_bwd.py

```diff
@@ -310,6 +323,58 @@ def test_learning_pipelining_without_interleaving_ucc_for_p2p(self):
     def test_inference_pipelining_without_interleaving_ucc_for_p2p(self):
         self._test_hybrid_backends(True)
 
+    @unittest.skipUnless(SUPPORT_ASYNC_BATCH_ISEND_IRECV, "Requires https://github.com/pytorch/pytorch/pull/82450")
+    def test_inference_pipelining_without_interleaving_async_batch_isend_irecv(self):
+        self._forward_backward_test_impl(
+            forward_only=False,
+            fwd_bwd_func=forward_backward_pipelining_without_interleaving,
+            pipeline_model_parallel_world_size=None,
+            virtual_pipeline_model_parallel_size=None,
+            async_comm=False,
+            default_backend=None,
+            p2p_backend=None,
+            sync_batch_comm=False,
+        )
+
+    @unittest.skipUnless(SUPPORT_ASYNC_BATCH_ISEND_IRECV, "Requires https://github.com/pytorch/pytorch/pull/82450")
+    def test_inference_pipelining_without_interleaving_async_batch_isend_irecv_infrerence(self):
```

Comment:
Perhaps `test_inference_pipelining_without_interleaving_async_batch_isend_irecv` would do it.